draft/overlay/RealTimeHAT.md

```diff
@@ -0,0 +1,83 @@
+<!--
+---
+name: RealTimeHAT
+class: board
+type: multi
+formfactor: HAT
+manufacturer: InnoRoute GmbH
+description: A sblabalbal board that can be used easily to show status of the Pi
```

Comment:
Is that a typo, or is `sblabalbal` a super-obscure word I've never heard of? :laughing:

Reply:
oops :D